test/rubygems/test_gem_ext_cargo_builder/rust_ruby_example/Cargo.toml

```diff
@@ -7,4 +7,4 @@ crate-type = ["cdylib"]
 
 [dependencies]
 # Needed until bindgen has the `allowlist_file` feature
```

Comment:
Is this comment still valid?

Reply:
yes it is, will update once it isn't, which is soon hopefully since it's merged into main!

Reply:
Updated

Reply:
Nice!